dotnet/Directory.Build.props

```diff
@@ -11,6 +11,11 @@
     <ImplicitUsings>disable</ImplicitUsings>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Once we're targeting only .NET 8+, this can be removed. -->
```

Comment:
We are building with .NET SDK 8.0+.  We're also currently building with .NET 6 and .NET 7 SDK.  If the suggestion is just substituting "building" for "targeting", ok.

Reply:
```suggestion
    <!-- Once we're only building with .NET SDK 8.0+, this can be removed.-->
```

Is this suggestion better?  "targeting" isn't right because you could targeting `net6.0`, but using the 8.0 SDK to build. The `IsAotCompatible` support came in the SDK - it doesn't matter what TFM you are targeting.

Reply:
Isn't there also a targeting component, though, in that the annotations aren't there in netstandard and so the additional analyzers this enables won't be relevant until the target also updates?

Reply:
Ah, but this is now just about ensuring IsTrimmable is set when the aot attribute isn't understood. Ok.